src/app/_components/Map/component.js

```diff
@@ -0,0 +1,17 @@
+import dynamic from 'next/dynamic';
+import PropTypes from 'prop-types';
+
+export function Map({ points, center, zoom }) {
+  const Window = dynamic(() => import('./window'), {
+    loading: () => <p>A map is loading</p>,
+    ssr: false,
+  });
+
+  return <Window points={points} center={center} zoom={zoom} />;
+}
+
+Map.propTypes = {
+  points: PropTypes.arrayOf(PropTypes.arrayOf(PropTypes.number)).isRequired,
```

Comment:
`points: PropTypes.arrayOf(PropTypes.arrayOf())`?
Shouldn't it be like `PropTypes.arrayOf(PropTypes.shape({ ... }))`?

Reply:
Thanks for comment! Yep, it should be in shape, fixed it.